Description: fix include-binaries automatically
 I want apt-now to be able to build new repos with bundled
packages again automatically. This should help.
 .
 apt-now (0.9-1) unstable; urgency=medium
 .
   * Initial release (Closes: #nnnn)  <nnnn is the bug number of your ITP>
Author: idk <idk@fireaxe.local>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: https://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: 2017-05-26

--- apt-now-0.9.orig/.fix-binaries.sh
+++ apt-now-0.9/.fix-binaries.sh
@@ -1,8 +1,9 @@
 #--dpkg-buildpackage-hook
+rm debian/source/include-binaries
 for p in $(find -name *.tar.*); do
-        echo $p | sed 's|./||'
+        echo $p | sed 's|./||'  | tee -a debian/source/include-binaries
 done
 
 for p in $(find -name *.deb); do
-        echo $p | sed 's|./||'
+        echo $p | sed 's|./||'  | tee -a debian/source/include-binaries
 done
